qiskit/execute.py

```diff
@@ -220,6 +237,19 @@ def execute(experiments, backend,
                             pass_manager=pass_manager,
                             )
 
+    # scheduling circuit into pulses.
+    if schedule_circuit:
+        if isinstance(experiments, Schedule) or \
+                (isinstance(experiments, list) and
+                 all(isinstance(e, Schedule) for e in experiments)):
```

Comment:
I don't think this supports mixed input, so this is sufficient:
```suggestion
        if isinstance(experiments, Schedule) or isinstance(experiments[0], Schedule):
```